framework/filters/Cors.php

```diff
@@ -97,8 +97,8 @@ class Cors extends ActionFilter
      */
     public function beforeAction($action)
     {
-        $this->request = $this->request ?: Yii::$app->getRequest();
-        $this->response = $this->response ?: Yii::$app->getResponse();
+        $this->request = $this->request ?: Yii::get('request');
+        $this->response = $this->response ?: Yii::get('response');
```

Comment:
Use `$action->controller->module->get('response')`

Reply:
This is risk prone, since filter instance may be applied for the module (application) level not only for controller.